tests/py/test_markdown.py

```diff
@@ -6,50 +7,99 @@
 
 class TestMarkdown(Harness):
 
-    # render
+    # r - render
```

Comment:
In past project in Java, we tried to avoid abbreviations in naming to increase readability.

Maybe rename the file to test_render and remove all 'render' string in the methods?